one_fm/overrides/leave_application.py

```diff
@@ -103,8 +103,16 @@ def close_todo(self):
     def on_submit(self):
         self.close_todo()
         self.close_shifts()
-        return super().on_submit()
-        self.db_set('status', 'Approved')
```

Comment:
Reason for removal of this lines

Reply:
I discovered that the return is calling the super submit function on leave application which there is a validation that is holding from approving as a reliever due to change in workflow. I actually copy all the functionality in the super submit function to the over ride function and I omit the validation in it

Reply:
![image](https://github.com/user-attachments/assets/0c82df68-b569-4e6b-8590-e611c1d43c07)